html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Document</title>
+  </head>
+  <body>
+  <h1>Coding Form!</h1>
+    <form action="www.codingform.com/why.php">
+      <p>
+        <label for="email">Email</label>
+        <input type="email" name="email"/>
+      </p>
+      <p>
+        <label for="coding-is-cool">Why is coding cool?</label>
```

Comment:
Same here, this is not configured correctly to pair with its form control element